### PR TITLE
corrects typos found during Vale testing

### PR DESCRIPTION
**Pull request by dawnkelly09**
I surfaced a couple of typos and things while screening this file with Vale.

### CODE REVIEW

File: docs/dev-tools/zombienet/overview.md [hunk 1]
```diff
@@ -387,7 +387,7 @@ Through the keyword `settings`, it's possible to define the general settings for
 - `image_pull_policy?` ++"string"++ - image pull policy to use in the network. Possible values are `Always`, `IfNotPresent`, and `Never`
 - `local_ip?` ++"string"++ - IP used for exposing local services (rpc/metrics/monitors). Defaults to `"127.0.0.1"`
 - `global_delay_network_global_settings?` ++"number"++ - delay in seconds to apply to the network
-- `node_verifier?` ++"string"++ - allow managing how to verify node readiness or disable by using `None`. Possible values are `None` and `Metric`. Defaults to `Metric`
+- `node_verifier?` ++"string"++ - specify how to verify node readniess or deactivate by using `None`. Possible values are `None` and `Metric`. Defaults to `Metric`
```
Comment (nhussein11) [inline docs/dev-tools/zombienet/overview.md:390]: I'm a bit puzzled by "deactivate" here. Did you set that as a rule? @dawnkelly09

**Comment by nhussein11** [comment]
For the rest, it LGTM. I'm seeing the potential of Vale here. Great job!! 🥳

File: docs/dev-tools/zombienet/overview.md [hunk 2]
```diff
@@ -157,9 +157,9 @@ It's important to note that each provider has specific requirements and associat
...
 
-Moreover, in order to create resources such as namespaces, pods, and cronJobs within the target cluster, you must have the appropriate permissions granted to your user or service account. These permissions are essential for managing and deploying applications effectively within Kubernetes.
+Moreover, in order to create resources such as namespaces, pods, and cron jobs within the target cluster, you must have the appropriate permissions granted to your user or service account. These permissions are essential for managing and deploying applications effectively within Kubernetes.
```
Comment (eshaben) [inline docs/dev-tools/zombienet/overview.md:162]: Isn't it CronJobs though? https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/

File: docs/dev-tools/zombienet/overview.md (hunk 1 above)
Comment (eshaben) [reply docs/dev-tools/zombienet/overview.md:390]: readniess - misspelled

Comment (dawnkelly09) [reply docs/dev-tools/zombienet/overview.md:390]: Readiness is my fault, I'll go fix it. 

@nhussein11 Yes, actually. Disabled is not recommended for inclusive language. This comes from the base Google Dev Docs styling.

**Review by eshaben — APPROVED** [review]
Sweet! Thanks! 🚀

**Pull request merged**